packages/core/forms/src/types/form-fields.ts

```diff
@@ -16,6 +18,7 @@ export interface FormFieldSchema extends Record<string, any> {
   hint?: string
   submitWhenNull?: boolean
   selectOptions?: Record<string, any>
+  component?: Component | null
```

Comment:
Wondering under what situation the schema would pass a `null` value to this field.

Reply:
Emm, like what we demonstrated in the `CatFoodField`. If we'd like to use a conditional check to flip the component, it would be better to use `null` instead of an implicit `undefined`

Reply:
Understood, resolved